src/Host/Broker/Impl/About/AboutController.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Management;
+using System.Reflection;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.R.Host.Broker.Interpreters;
+using Microsoft.R.Host.Broker.Security;
+using Microsoft.R.Host.Protocol;
+using Microsoft.R.Interpreters;
+
+namespace Microsoft.R.Host.Broker.About {
+    [Authorize(Policy = Policies.RUser)]
+    [Route("/about")]
+    public class AboutController : Controller {
+        private readonly InterpreterManager _interpManager;
+
+        public AboutController(InterpreterManager interpManager) {
+            _interpManager = interpManager;
+        }
+
+        [HttpGet]
+        public AboutHost Get() {
+            var a = new AboutHost();
+
+            a.Version = Assembly.GetExecutingAssembly().GetName().Version;
```

Comment:
Can you put all this logic in a separate non-HTTP-aware class, and call it from here? I'd like controllers to be kept separate from the rest of it to simplify testing.